viral/run_oasis.py

```
@@ -0,0 +1,169 @@
+            _, ax1 = plt.subplots()
+            baseobj = BaselineRemoval(cell)
+            cell_baselined = baseobj.ZhangFit()
+
+            residual = np.sum(np.abs(cell - cell_baselined))
+
+            print(f"residual is {residual}")
+
+            # ax1.plot(cell, color="pink")
+            ax1.plot(cell_baselined, color="blue")
+            ax1.plot(spikes, color="black")
+            ax2 = ax1.twinx()
```

Ruff (0.8.2)

Local variable `ax2` is assigned to but never used

Remove assignment to unused variable `ax2`

(F841)